### DISCUSSION TITLE
新手小白,请问下怎么处理大量数据下组件渲染的问题啊,如图

### DISCUSSION THREAD

**Discussion by xnfy520** [element-plus/element-plus#3123] [Q&A]
<img width="1210" alt="截屏2021-08-30 上午11 08 52" src="https://user-images.githubusercontent.com/2343777/131279701-667af87d-e983-41e9-b25b-9859600201c4.png">

**Comment by jw-foss** [comment]
Can you be more specific about your problem here?
